README.md

```diff
@@ -9,24 +9,48 @@ Branch Policy
-is created from master, and the new branch goes into stabilization/release phase.
-- Major numbers are stepped up every new release.  The release schedule is
-driven by a time plan.  Use [semantic versioning](http://semver.org/) for
-minor and patch numbers.
-- GENIVI-specific release tags, such as "M-0.2" for the second baseline
-*pre*release for the "M" platform may also be added.
+  (PRs) should be proposed as changes to master, unless you know they are
+  applicable to a particular release only.
+- Major numbers are stepped up every new release of the GENIVI Compliance
+  Specification.  The release schedule is driven by a time plan. Use [semantic
+  versioning](http://semver.org/) for minor and patch numbers.
+- The GENIVI Compliance specification is released annually, while meta-ivi is
+  released bi-annually meaning that we get a gap. To keep the major version
+  numbers in sync, we will release 14.0.0 together with the Compliance
+  Specification and 14.50.0 half way to 15.0.0. 
+  - Note: Updates to the 14.0.0 release would then become 14.0.1 and/or 14.1.0, which
+    can be released __after__ 14.50.0.
+- Somewhere near a new major release of the GENIVI Compliance specification, a
+  numbered release branch (e.g. 14.0) is created from master, and the new
+  branch goes into stabilization/release phase.
+- In the middle between the GENIVI Compliance Specification version 14.0 and
+  15.0 a numbered branch is created for the 14.50 meta-ivi release.
 - After a versioned release, the release branch remains for maintenance and
-updates.
+  updates.
 - The project maintainer shall ensure that relevant patches are
-cherry-picked to every branch where they apply. I.e. patches should be
-back-ported *at minimum* to the Support Window versions as defined below.
+  cherry-picked to every branch where they apply. I.e. patches should be
+  back-ported *at minimum* to the Support Window versions as defined below.
 - In general, prefer a linear commit history (rebase and cherry-pick),
   applying merge commits only where absolutely necessary to sort out a
   complex merge situation (which we should rarely have).
 
+Tag Policy
+----------
+- GENIVI releases use release names called P-0.1, P-0.2, P-1.0 indicating
+  levels of maturity, where P signifies the release name, 0.1 indicates that
+  it's a pre-release and is work in progress, 0.2 signifies that the Compliance
+  Specification should be at a point where the fundamentals are in place and
+  1.0 is a refinement on 0.2 where the specifications are finalized. 
+  - P-0.1 is equivalent to 14.0.0-rc.1
+  - P-0.2 is equivalent to 14.0.0-rc.2
+  - P-1.0 is equivalent to 14.0.0 and should be the final release. 
+- Meta-ivi creates tags with the GENIVI release naming scheme upon releasing
+  new versions.
+- Meta-ivi creates tags based on the version number in accordance with this
+  policy.
+- Meta-ivi creates a Github release based on the version number in accordance
```

Comment:
Also this, good to keep.  The text is a bit repetitive.  I think you can just combine it to be a bit less clunky.

Proposal:  -- "creates a tag and a corresponding GitHub release..." 
"named according to the version policy"

Reply:
Fixed.